src/main/java/org/opentripplanner/graph_builder/module/DirectTransferGenerator.java

```diff
@@ -112,10 +102,7 @@ public void buildGraph(
                         // Skip the origin stop, loop transfers are not needed.
                         if (sd.stop == stop) { continue; }
                         if (sd.stop instanceof Stop) { continue; }
-                        distinctTransfers.put(
-                            new TransferKey(sd.stop, stop, sd.edges),
-                            new PathTransfer(sd.stop, stop, sd.distance, sd.edges)
-                        );
+                        updateDistinctTransfers(distinctTransfers, stop, transferProfile.modes.transferMode, sd);
```

Comment:
This has the parameters reversed (`sd.stop, stop`) compared to the previous occurrence (`stop, sd.stop`).

On a first pass it doesn't look like `updateDistinctTransfers()` takes that into account.